data/question_mappings.csv

```diff
@@ -0,0 +1,185 @@
+source_form,source_field,domain,concept_id,concept_name,value_variable,comment

```

Comment:
Are you using this file in the reports?

Reply:
Yes, to get the source concept ids for measurements.

Reply:
Concept Ids are added to the Measure bounds CSV.